src/org/omegat/Bundle.properties

```
@@ -3011,3 +3011,14 @@ THEME_OMEGAT_DEFAULT_NAME=OmegaT Default
 THEME_OMEGAT_DEFAULT_DESCRIPTION=OmegaT default theme
 THEME_OMEGAT_CLASSIC_NAME=OmegaT Classic
 THEME_OMEGAT_CLASSIC_DESCRIPTION=OmegaT classic theme
+
+GPG_EXTERNAL_SIGNER_BUFFER_ERROR=Error while getting output of external program
+GPG_EXTERNAL_SIGNER_CANNOT_SEARCH=Cannot find GPG executable on the bash $PATH
```

PATH is not a bash-specific concept, so this message seems odd. Did it also come from EGit?

```suggestion
GPG_EXTERNAL_SIGNER_CANNOT_SEARCH=Cannot find GPG executable
```

---

I cannot remember but I might pick it from EGit.